lib/utils/globals.dart

```diff
@@ -0,0 +1,5 @@
+import 'package:flutter/material.dart';
+
+class Globals {
+  static GlobalKey<ScaffoldState> scaffoldKey = GlobalKey<ScaffoldState>();
```

Comment:
Why? Remember that keys must be unique.
I don't know if there's a need to create a class for this, because it won't be reused at other times. 💭

Reply:
Now I understanding this better, after [read this](https://stackoverflow.com/a/50050097/13096514).